resources/views/posts/manage.blade.php

```diff
@@ -33,7 +33,7 @@ class="fa-solid fa-pen-to-square"
                         Show</a>
                 </td>
                     
-                    @if (auth()->user()->role_id == $post->user->role_id)
+                    @if (auth()->user()->role_id == $post->user->role_id || auth()->user()->role_id == 1 )
```

Comment:
you can move this check to the policy so if it returns false that page won't render, or if you have only that single component to visible or not based on the user type you can create a function inside your user:odel for eg a function called ` isAdmin(){return $this->role_id == 1  }`  to check if the auth user is owner or not and another one that will return true if the auth is owner and if not it will check if it is the post owner `isPostOwner( return $this->isAdmin() ? true : $this->role_id == $post->user->role_id )` and instead of doing this `@if (auth()->user()->role_id == $post->user->role_id || auth()->user()->role_id == 1 )` do this `@if (auth()->user()->isPostOwner() )`